edx_repo_tools/dependabot_yml.py

```diff
@@ -48,15 +67,18 @@ def modernize(self):
 @click.option(
     '--path', default='.github/dependabot.yml',
     help="Path to target dependabot.yml file")
-def main(path):
+@click.option(
+    '--reviewer', default=None,
+    help="Name of the reviewer")
```

Comment:
Kindly add the info that the reviewer should be a github username or team name, and the team name should be in the format of `org-name/team-name`.

Reply:
sure